### PR TITLE
This will import our custom mounting file instead of enzyme

### PR DESCRIPTION
**Pull request by pvienneau**
- Added path traversal logic to determine depth of component relative to `lib/` folder in Particle
- Importing custom mounts

### CODE REVIEW

**Review by devriestyler — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ function getTemplate(name) {
     return fs.readFileSync(path.join(__dirname, 'stubs', `${name}.stub`)).toString('utf8');
 }
 
-function renderTemplate(template, params) {
+function renderTemplate(template, params) {
```
Comment (ericadamski) [inline src/index.js]: this has trailing spaces

**Review by ericadamski — CHANGES_REQUESTED** [review]

File: dist/index.js [hunk 2]
```diff
@@ -155,14 +155,12 @@ function createNewFile(source, output, name, interactions) {
...
-    var folders = path.parse(source).dir.split(path.sep).reverse();
-
-    for (var ii = 0; ii < folders.length; ii++) {
-        logger.log(folders[ii]);
-        if (folders[ii] == 'lib') break;
-
-        relativeLibPath += '../';
+    var relativeLibPath = './';
+    try {
+        fs.accessSync(path.resolve('lisdf'));
```
Comment (ericadamski) [inline dist/index.js]: Pretty sure this isn't suppose to be 'lisdf'

File: dist/index.js [hunk 3]
```diff
@@ -153,7 +154,16 @@ function createNewFile(source, output, name, interactions) {
...
     logger.skip();
     var relativePath = path.join(path.relative(path.resolve(path.dirname(output)), path.dirname(source)), path.basename(source));
-    var code = [createDescribeShell(name, name, relativePath), createRenderTestShell(name), createSubComponentTestShell(name), createDefaultPropTypesShell(name), createPropTypesShell(name)];
+
+    var relativeLibPath = './';
+    try {
+        fs.accessSync(path.resolve('lisdf'));
+        relativeLibPath = path.relative(source, path.resolve('lib/components')) + '/';
+    } catch (e) {
+        ; //do nothing
```
Comment (ericadamski) [inline dist/index.js]: There is no need for the ';' do you like it style wise?

File: dist/stubs/describe-shell-start.stub [hunk 4]
```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 
-import { shallow, mount } from 'enzyme';
 import injectTapEventPlugin from 'react-tap-event-plugin';
 
 import <%= className %> from '<%= relativePath %>';
+import { render, shallow, mount } from '<%= relativeLibPath %>utils/enzyme/mounting.js';
```
Comment (ericadamski) [inline dist/stubs/describe-shell-start.stub]: Looking at this I think the ´/´ should be here.

**Review by ericadamski — CHANGES_REQUESTED** [review]
Just a few small things :)

**Review by ericadamski — APPROVED** [review]

**Pull request merged**